solutions/pipelines-review/pipeline.yaml

```
@@ -57,18 +63,24 @@ spec:
           name: maven_config
 
     - name: oc-deploy
-      runAfter:
-        - "build"
       taskRef:
-        name: openshift-client
-        kind: ClusterTask
+        resolver: cluster
+        params:
+          - name: kind
+            value: task
+          - name: name
+            value: openshift-client
+          - name: namespace
+            value: openshift-pipelines
+      runAfter:
+        - build
       workspaces:
-        - name: manifest-dir
+        - name: manifest_dir
           workspace: shared
       params:
         - name: SCRIPT
           value: |
-            cd  "$(workspaces.manifest-dir.path)/$(params.MVN_APP_PATH)" || exit 1
+            cd  "$(workspaces.manifest_dir.path)/$(params.MVN_APP_PATH)" || exit 1
             oc new-build --name=$(params.DEPLOY_APP_NAME) \
               -l app=$(params.DEPLOY_APP_NAME)  --binary=true \
```

Changing only to the `diff` command between this file and the student file does not show any changes

```suggestion
              -l app=$(params.DEPLOY_APP_NAME) --binary=true \
```